### PR TITLE
update migration discovery information

### PR DESCRIPTION
**Pull request by shimks**
### Description

- connected to https://github.com/strongloop/loopback.io/issues/284

#### Related issues

- connect to <link_to_referenced_issue>

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: available-connectors.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "settings": {
       "localStorage": {
         "type": "string",
-        "description": "window.localStorage key to use for persistence (browser only)"
+        "description":
+          "window.localStorage key to use for persistence (browser only)"
```
Comment (bajtos) [inline available-connectors.json]: Please revert all white-space-only changes. Did you use Prettier to format this file? It formats JSON files differently than `JSON.stringify` does :( But! The recently-released Prettier version 1.13 support json-stringify format, see https://prettier.io/blog/2018/05/27/1.13.0.html#added-json-stringify-parser-for-jsonstringify-style-formatting-4450-by-ikatyang.

File: available-connectors.json [hunk 2]
```diff
@@ -69,13 +70,14 @@
...
     "features": {
-      "discovery": false,
+      "discovery": true,
       "migration": false
     },
     "settings": {
       "dsn": {
         "type": "string",
-        "description": "Connection String dsn to override other settings (eg: DATABASE=MY_DB;HOSTNAME=MY_HOST;PORT=MY_PORT;PROTOCOL=TCPIP;UID=MY_UID;PWD=MY_PWD))"
+        "description":
```
Comment (bajtos) [inline available-connectors.json]: Unrelated white-space change, please revert here and in all other places below too.

**Comment by shimks** [comment]
@slnode test please

**Comment by shimks** [comment]
@slnode test please

**Pull request merged**